lib/components/form/batch-settings.tsx

```diff
@@ -222,15 +233,19 @@ class BatchSettings extends Component<Props, State> {
               id: 'components.BatchSettings.planTripTooltip'
             })}
           >
-            <Icon
-              className="fa-2x"
-              type={
-                hasValidLocation(currentQuery, 'from') ||
-                hasValidLocation(currentQuery, 'to')
-                  ? 'refresh'
-                  : 'search'
-              }
-            />
+            <StyledIconWrapper
+              noMargin
+              size="2x"
+              static
+              style={{ height: '100%', width: '100%' }}
```

Comment:
This icon still isn't looking right. Did the font size fix somehow get removed?

Reply:
Yes, I think it got lost in the great git revert. oops